server/test/article-search.spec.js

```diff
@@ -0,0 +1,35 @@
+import chai, { expect } from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../../index';
+
+
+chai.use(chaiHttp);
+describe('POST /api/v1/article/?query', () => {
```

Comment:
Please make corrections to the descriptions in the describe block, at `POST` should be `GET`

Reply:
Alright..thanks